docs/api.md

```diff
@@ -73,6 +73,26 @@ const logger = pino({
 logger.foo('hi')
 ```
 
+<a id=opt-useOnlyCustomLevels></a>
+#### `useOnlyCustomLevels` (Boolean)
```

Comment:
For all options that drastically change the output we should be adding a warning that using the option can break compatibility with downstream transports. See https://github.com/pinojs/pino/blob/7dd98cc8c2c41e1458e9eb6e0de07c18abb2ac9e/docs/api.md#uselevellabels-boolean for an example.

Sorry for the late review. Been very busy.